test/hotspot/jtreg/vmTestbase/nsk/jdi/VirtualMachine/exit/exit001.java

```diff
@@ -173,6 +173,12 @@ private int runThis(String argv[], PrintStream out) {
             logHandler.complain("TEST FAILED");
         }
 
+        int code = debuggee.waitFor();
+        if (code != 0) {
```

Comment:
Why do the other tests check for Consts.JCK_STATUS_BASE, but here we check for 0?